netflix_utils.py

```diff
@@ -87,9 +87,9 @@ def parse_args(self, argv):
         parser = argparse.ArgumentParser(description="""
         Simple matrix factorization algorithm to find latent variables that represents netflix prize data.
         """, formatter_class=argparse.ArgumentDefaultsHelpFormatter)
-        parser.add_argument('-i', '--input', default = 'nf_prize_dataset/nf_probe_training.npy',
+        parser.add_argument('-i', '--input', default = 'nf_prize_dataset/nf_training.npz',
```

Comment:
Just to be sure, we can load a npz the same way we previously loaded npy?

Reply:
Yep, au final ça marche pareil (faut juste regénérer les fichiers)